infoblox_client/objects.py

```diff
@@ -731,13 +732,15 @@ class DhcpOption(SubObjects):
 
 class IPRange(InfobloxObject):
     _fields = ['start_addr', 'end_addr', 'network_view',
-               'network', 'extattrs', 'disable']
+               'network', 'extattrs', 'disable', 'server_association_type',
+               'failover_association']
```

Comment:
According to the wapi document,
The basic version of the object contains the field(s): comment, end_addr, network, network_view, start_addr. 
Hence, i feel we should use _return_field override instead of making these fields as basic fields to the class.

Reply:
@adsri , please see my comment below (https://github.com/infobloxopen/infoblox-client/pull/187#discussion_r312081867)